src/dispatch/incident/flows.py

```diff
@@ -542,6 +542,58 @@ def set_conversation_topic(incident: Incident, db_session: SessionLocal):
         log.exception(e)
 
 
+def set_conversation_bookmarks(incident: Incident, db_session: SessionLocal):
+    """Sets the conversation bookmarks."""
+    if not incident.conversation:
+        log.warning("Conversation bookmark not set. No conversation available for this incident.")
+        return
+
+    plugin = plugin_service.get_active_instance(
+        db_session=db_session, project_id=incident.project.id, plugin_type="conversation"
+    )
+    try:
```

Comment:
We may want to check if `plugin` exists here, and if not, log a warning and return (see https://github.com/Netflix/dispatch/pull/2784/files#diff-9f1e8b67dd91333ce78c91db2036e284006cf54fa05595c9e43b0b66451d628fR122 as an example).